Common subdirectories: ipw2200-1.1.2/in-tree and ipw2200-1.1.2-cflags/in-tree
diff -u ipw2200-1.1.2/Makefile ipw2200-1.1.2-cflags/Makefile
--- ipw2200-1.1.2/Makefile	2006-04-01 03:47:23.000000000 +0200
+++ ipw2200-1.1.2-cflags/Makefile	2006-04-04 17:28:26.000000000 +0200
@@ -84,20 +84,20 @@
 EXTRA_CFLAGS += -DCONFIG_PM
 
 ifeq ($(EXTERNAL_BUILD),y)
-ifdef CONFIG_IPW2200_DEBUG	
-EXTRA_CFLAGS += -DCONFIG_IPW2200_DEBUG=$(CONFIG_IPW2200_DEBUG)
+ifeq ($(CONFIG_IPW2200_DEBUG),y)
+EXTRA_CFLAGS += -DCONFIG_IPW2200_DEBUG
 endif
-ifdef CONFIG_IPW2200_MONITOR
-EXTRA_CFLAGS += -DCONFIG_IPW2200_MONITOR=$(CONFIG_IPW2200_MONITOR)
-ifdef CONFIG_IPW2200_RADIOTAP
-EXTRA_CFLAGS += -DCONFIG_IPW2200_RADIOTAP=$(CONFIG_IPW2200_RADIOTAP)
+ifeq ($(CONFIG_IPW2200_MONITOR),y)
+EXTRA_CFLAGS += -DCONFIG_IPW2200_MONITOR
+ifeq ($(CONFIG_IPW2200_RADIOTAP),y)
+EXTRA_CFLAGS += -DCONFIG_IPW2200_RADIOTAP
 endif
-ifdef CONFIG_IPW2200_PROMISCUOUS
-EXTRA_CFLAGS += -DCONFIG_IPW2200_PROMISCUOUS=$(CONFIG_IPW2200_PROMISCUOUS)
+ifeq ($(CONFIG_IPW2200_PROMISCUOUS),y)
+EXTRA_CFLAGS += -DCONFIG_IPW2200_PROMISCUOUS
 endif
 endif
-ifdef CONFIG_IPW2200_QOS
-EXTRA_CFLAGS += -DCONFIG_IPW2200_QOS=$(CONFIG_IPW2200_QOS)
+ifeq ($(CONFIG_IPW2200_QOS),y)
+EXTRA_CFLAGS += -DCONFIG_IPW2200_QOS
 endif
 endif
 
